extruct/__init__.py

```diff
@@ -8,29 +8,29 @@
 from extruct.xmldom import XmlDomHTMLParser
 
 logger = logging.getLogger(__name__)
-VALID = ('microdata', 'opengraph', 'jsonld', 'microformat', 'rdfa')
+VALID = ('microdata', 'opengraph', 'json-ld', 'microformat', 'rdfa')
 
 def extract(htmlstring, url='http://www.example.com/', encoding="UTF-8",
-            syntaxes=['microdata', 'opengraph', 'jsonld', 'microformat', 'rdfa'],
+            syntaxes=['microdata', 'opengraph', 'json-ld', 'microformat', 'rdfa'],
```

Comment:
can ``VALID`` constant be used here and below, and in tool.py, where we need a list of supported microformats?